src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -171,6 +171,7 @@ def failure_count_transaction(org_id, metric_ids, alias=None):
     return _dist_count_aggregation_on_tx_status_factory(
         org_id,
         exclude_tx_statuses=[
+            # See statuses in https://docs.sentry.io/product/performance/metrics/#failure-rate
```

Comment:
Coming from https://github.com/getsentry/sentry/pull/33125#discussion_r841824432. It isn't really related to this PR, but I'm lazy enough to not open another PR just for this comment.